app/uk/gov/hmrc/pillar2/service/RegistrationService.scala

```diff
@@ -101,7 +101,7 @@ class RegistrationService @Inject() (
       ultimateParentEntityName = registerWithoutIDRequest.organisation.organisationName,
       addressLine1 = registerWithoutIDRequest.address.addressLine1,
       addressLine2 = registerWithoutIDRequest.address.addressLine2.getOrElse(""),
-      townOrCity = registerWithoutIDRequest.address.addressLine3,
+      townOrCity = registerWithoutIDRequest.address.addressLine3.getOrElse(""),
```

Comment:
@TevynAllen  I am more worried about this tbh

Reply:
Why? Its for the audit event